src/sentry/hybridcloud/models/orgauthtokenreplica.py

```diff
@@ -35,6 +35,7 @@ class OrgAuthTokenReplica(Model):
     class Meta:
         app_label = "hybridcloud"
         db_table = "hybridcloud_orgauthtokenreplica"
+        indexes = (models.Index(fields=["token_hashed"]),)
```

Comment:
@markstory should `date_deactivated` be part of this index as well? I notice we use it in the query.

Reply:
I don't think it would change query plans much as the token column has unique contents currently.